client/core/account.go

```diff
@@ -2,11 +2,54 @@ package core
 
 import (
 	"encoding/hex"
+	"fmt"
 
 	"decred.org/dcrdex/client/db"
 	"github.com/decred/dcrd/dcrec/secp256k1/v3"
 )
 
+// AccountDisable is used to disable an account by given host and application
+// password.
+func (c *Core) AccountDisable(pw []byte, host string) error {
+	// Validate password.
+	_, err := c.encryptionKey(pw)
+	if err != nil {
+		return codedError(passwordErr, err)
+	}
```

Comment:
Just want to point out that validating the password is kinda superficial. One could comment out this code, compile, and get past that.

Reply:
Thought the same, any easy way to improve that ?

Reply:
I think the alternative is to not require the password.  Which should be OK if the auth middleware is working correctly, and cookies aren't stolen, and on the RPC server side they've auth'd properly there.
But we also request the app password for certain actions that are potentially irreversible, such as this one.  I'm ok either way for this function.

Reply:
Actually, this would stop a remote attacker, as they may not be able to swap your executable, so please leave it.

Reply:
Yep. 
Weak protection better than no protection.

Reply:
I just want to point out that it's the job of say the RPC server (rpcuser/pass) and webserver (auth cookie) to deal with permissions, not the Core Go API, although requiring a password to make this harder is an existing pattern so I don't mind it staying here.